docs/description.md

```diff
@@ -1,35 +1,36 @@
-# BUILD A PRODUCT: Teamwork
+# PRODUCT NAME: TableSurfing
 
 ## Project Overview
 
-Teamwork is an ​ internal social network for employees of an organization. The goal of this
-application is to facilitate more interaction between colleagues and promote team bonding.
+TableSurfing is a social network for a freelancers and guests interested in office or table sharing. The goal of this application is to facilitate networking and interaction between freelancers and their host.
 
 ## Project Timelines
 
-Total Duration​: 5 weeks
+Total Duration​: 6 weeks
 
 ## Required Features
 
-1. Admin can create an employee user account.
-2. Admin/Employees can sign in.
-3. Employees can post gifs.
-4. Employees can write and post articles.
-5. Employees can edit their articles.
-6. Employees can delete their articles.
-7. Employees can delete their gifs post.
-8. Employees can comment on other colleagues' article post.
-9. Employees can comment on other colleagues' gif post.
-10. Employees can view all articles and gifs, showing the most recently posted articles or gifs
-    first.
-11. Employees can view a specific article.
-12. Employees can view a specific gif post.
+1. Host/guest can create an account.
+2. Host/guest can sign in.
+3. Host can post their available space.
+4. Guest can find an available space nearby with a glance.
+5. Host can view the guest profile.
+6. Guest ca view the host profile.
+7. Host can delete occupied listing.
+8. Guest can search by city or name a location.
+9. Guest and host can chat.
+10. Guest can get the direction via a map.
+11. Host can accept or decline a request.
+12. Guest can view a specific listing.
+13. Guest can book a space.
+14. Guest can post gifs.
```

Comment:
Let's remove gifs? I'm not sure they need them in their chat.

Reply:
Yeah, we will keep it simple.